### DISCUSSION TITLE
Looking for Old compiled releases that work in mac mini mid 2012 High Sierra

### DISCUSSION THREAD

**Discussion by egpgit** [dirkwhoffmann/virtualc64#726] [Q&A]
Hi everybody,
Years ago I installed version 1.0rc2 in my mac mini, now I want to update it to latest version which works in High Sierra in mac with no Metal GPU (mac mini mid 2012).
Where can I find it? The official Website only has the latest version compiled to download other websites links with macpkg.icu where there are only advertisements but I can't download anything.

Thanks and sorry for my poor english

**Comment by dirkwhoffmann** [comment]
It's buried deeply in the repo. If you really want to resurrect this very old version (I wouldn't recommend it), you can get the sources from here:

https://github.com/dirkwhoffmann/virtualc64/tree/f589995186f14d626f16d2e22fce6af443f075ab

There is no executable, so you would have to compile it yourself (with a decently old Xcode version). Maybe someone else still has the executable around somewhere.

**Comment by mithrendal** [comment]
Hi @egpgit, I have found a couple of old ready to install versions on csdb.dk

https://csdb.dk/search/?seinsel=all&search=virtualc64&Go.x=0&Go.y=0

hope that helps you
